### DISCUSSION TITLE
How can I embed an external file?

### DISCUSSION THREAD

**Discussion by nadrojisk** [mayurankv/Obsidian-Code-Styler#273] [Q&A] [answered]
I noticed in the documentation that there is support for embedding files. However, I cannot seem to get it working correctly? Any help would be appreciated! You can see the file is located in the same directory with it. I believe I just have the syntax wrong.

<img width="544" alt="image" src="https://github.com/mayurankv/Obsidian-Code-Styler/assets/37343577/b91b23fc-04e3-4bca-af36-e382b894e451">
<img width="336" alt="image" src="https://github.com/mayurankv/Obsidian-Code-Styler/assets/37343577/134f15fe-b53d-48a1-92e7-ce791fc36b39">

<img width="197" alt="image" src="https://github.com/mayurankv/Obsidian-Code-Styler/assets/37343577/6875a351-4115-46d4-aee5-909eb7ed3180">

**Comment by mayurankv** [comment, marked as answer]
Sorry! The YAML should be inside the code block, like this:

````
```reference
file: test.py
```
````

**Reply by mayurankv** [reply]
(I really need to clean up the documentation about this!)

**Reply by nadrojisk** [reply]
Oh shoot I tried this earlier and got a similar error which is when I gave up 😂 .

```reference
file:test.py
```

Thank you!

**Reply by mayurankv** [reply]
Oh hmmm, im not at my laptop to check but maybe it needs to be `file:[[test.py]]`, one of those two should definitely work!

**Reply by nadrojisk** [reply]
Oh no it's just because there needs to be a space I believe after the `:` otherwise YAML can't parse correctly. I've got it working now with your suggestion
